kubernetes-versions.json

```diff
@@ -0,0 +1,7 @@
+[
+    "1.23",
+    "1.24",
+    "1.25",
+    "1.26",
+    "1.27"
+]
```

Comment:
At some point I'd like to use this in lieu of the hard-coded k8s versions in the Makefile.